UI/ui_mainmode.csv

```diff
@@ -3140,7 +3140,7 @@ SynchronizeMyParty#0,"""Your party"""
 SynchronizeMemberName#0,"""Member Name"""
 SynchronizeStageClear#0,"""Cleared Stage"""
 SynchronizeQuestName#0,"""Quest name"""
-SynchronizeBossTitle#0,"""Teleporter"""
+SynchronizeBossTitle#0,"""teleporter"""
```

Comment:
All changed instances of "teleporter" in this file need to be capitalized, the unchanged ones are fine